test/Testing/LeanCode.IntegrationTestHelpers.Tests/TestApp.cs

```diff
@@ -7,13 +7,17 @@
 using Microsoft.AspNetCore.TestHost;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Serilog.Events;
 
 namespace LeanCode.IntegrationTestHelpers.Tests;
 
 public class TestApp : LeanCodeTestFactory<App.Startup>
 {
-    protected override ConfigurationOverrides Configuration { get; } =
-        new("SqlServer__ConnectionStringBase", "SqlServer:ConnectionString", Serilog.Events.LogEventLevel.Error, false);
+    protected override TestConnectionString ConnectionStringConfig { get; } =
+        new("SqlServer__ConnectionStringBase", "SqlServer:ConnectionString");
```

Comment:
Don't we need the same test for postgres?

Reply:
This test suite tests if the helpers itself work. They don't test integration with any particular database (they need _a_ database, but it is not important).